web-console/developer/builds/builds.md

```diff
@@ -1,7 +1,22 @@
 # Builds
 The Builds page is a primary nav item in the OpenShift Developer perspective.
 
-Design documentation for the Builds section can be found [here](https://docs.google.com/document/d/1MWUJ9I_YpI-4e4RVM89g7VNKAj-VTrffDl245IglTfs/edit?usp=sharing)
+## Overview
 
-## Note
-Links to google documents are temporary until the documentation is converted to markdown files and added to this design repository.
\ No newline at end of file
+The Builds page is different from the Administrator perspective.  Clicking on the Builds navigation item will display Build Config resources.
+
+## Build Config Resources
+
+![Add build config image](img/image1.png)
+
+## Build Config Details - Overview Tab
+
+![Add build config image](img/image2.png)
+
+## Build Config Details - Builds 
+
+![Add build config image](img/image3.png)
+
+## Build Details - Overview Tab
+
+![Add build config image](img/image4.png)
```

Comment:
Should this link back to the Build Config in the breadcrumbs?

Reply:
based on the conventions from OpenShift admin side, i think that the parent in the breadcrumb always brings you to a resource table of those objects themselves & doesn't match the nav item.  @beanh66 can you confirm ?

Reply:
I agree @lizsurette this breadcrumb feels wrong. The OpenShift admin side is different because we have a nav item for both Build Configs and Builds but on the dev side only one nav item. In this case you can only drill into Builds from the Build Config Details so I think the breadcrumb would be "Build Configs  >  (BC) httpd-example  >  (B) httpd-example-1"